orca-queue/src/main/kotlin/com/netflix/spinnaker/orca/q/handler/ExpressionAware.kt

```diff
@@ -35,6 +38,9 @@ interface ExpressionAware {
   val contextParameterProcessor: ContextParameterProcessor
   val log: Logger
     get() = LoggerFactory.getLogger(javaClass)
+  private val mapper: ObjectMapper
+    get() = OrcaObjectMapper.newInstance()
```

Comment:
mmm, do we really want to `new` up a new object mapper each time? i think just assigning this once would be better

Reply:
yeah, my bad, thanks!